pom.xml

```diff
@@ -204,6 +204,39 @@
                 <artifactId>alfresco-events</artifactId>
                 <version>1.2.10</version>
             </dependency>
+            <dependency>
+                <groupId>org.alfresco.surf</groupId>
+                <artifactId>spring-surf-core-configservice</artifactId>
+                <version>${dependency.alfresco-spring-webscripts.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.alfresco.surf</groupId>
+                <artifactId>spring-webscripts</artifactId>
+                <version>${dependency.alfresco-spring-webscripts.version}</version>
+                <exclusions>
+                    <exclusion>
```

Comment:
Perhaps better to pull in stuff that is required and just exclude all transitive dependencies here.